README.md

```diff
@@ -67,6 +67,13 @@ If you want correct CPU temperature displayed in `byobu` status notifications, a
 
     MONITORED_TEMP=/sys/class/hwmon/hwmon1/temp1_input
 
+## Display
+
+### Fractional Scaling
+On KDE, Deepin and other Qt-based desktop environments, fractional scaling is supported without any hassle but this is not the case for GTK-based desktop environments such as GNOME and XFCE. Using Wayland can be an option, but the apps that don't support Wayland natively will still have issues such fuzziness all over the window. A workaround for Xorg allows for 150% scaling without any tear at the const of a little performance drop:
```

Comment:
s/const/cost/ :)

Reply:
Oh lol seems pretty natural for me. Changed.